theme/yarn.lock

```diff
@@ -197,6 +197,13 @@
   dependencies:
     purgecss "^4.0.3"
 
+"@growthbook/growthbook@^1.2.1":
```

Comment:
In theory, your above markdown changes should not require a update to `
yarn.lock`. Any idea why this is here? Can we remove changes from the commit? (e.g. if we need to bump library versions, that should be done in a separate PR and investigated with depth separately from a content update)

Reply:
Hey Troy, that change in the yarn.lock showed up after running the "make ensure" command so I remove that change if it shouldn't be included